### ISSUE TITLE
URGENT- Add the tissue only button back to the standard uncustomized search screen 

### ISSUE THREAD

**Issue by atrox10** [ArctosDB/arctos#8350]
We REALLY need to have the tissue-only button on the standard search screen that comes up, without customization. This got removed at some point and needs to be put back. Most outside Arctos users NEED the tissue only button and don’t know how to customize and don’t need a login, please add this back (otherwise they go to vertnet which has old data but is an easier interface)
Thanks!)

**Comment by atrox10** [comment]
and I mean the "is tissue?" button, thats what I am referring to. Its probably one of the common searchers for outside Arctos folks/public, that they are looking for tissues for loans from/ in our collections.

**Comment by mkoo** [comment]
This is on the AWG agenda for Thursday

**Comment by dustymc** [comment]
If that audience fails that spectacularly, maybe the other ~95% of users do too and we should default to giving everyone [everything](https://arctos.database.museum/search.cfm?sp=OhDiosMisOjos)? I've always thought that such a starting point would be completely overwhelming to most users, and there are a [lot](https://arctos.database.museum/search.cfm?materials=dna&is_tissue=1&sp=OhDiosMisOjos) of ways to get lost in such a mass of never-used-together options, plus many more that time out ([help request](https://github.com/ArctosDB/internal/issues/330)) with literally everything turned on, but that's still possibly less-evil than wandering off to vertnet??

Could this be addressed by a customized search URL from your web page (or wherever these users are coming from)? A nice example of that: https://arctos.database.museum/uamart

Input from the less tissue-centric collections, here or [there](https://docs.google.com/document/d/1820m9g7z3vnBSkAIxi39awJCLX_EEQgVD-_VwjxgGWg/edit?tab=t.0), would be useful.



 @keg34
 @leet1984
 @javanveldhuizen
 @sjshirar
 @ccicero
 @mkoo
 @mvzhuang
 @AJLinn
 @KatherineLAnderson
 @DellaCHall
 @aklompma
 @droberts49
 @wellerjes
 @jrpletch
 @Nicole-Ridgwell-NMMNHS
 @brandon-s-thompson
 @ehalverson26
 @kat-sterner
 @jessicatir
 @ufarrell
 @ronaldeng
 @WaigePilson
 @KatelinCross

**Comment by mkoo** [comment]
Discussed yesterday (were we in AWG or CT meeting?!)
Anyway, we decided we can create a new preset for now so sharing that search URL will be easy (can be linked on our MVZ website for example)
Here's what it would look like: https://arctos.database.museum/search.cfm?sp=Tissue

It's very simple with the Tissue checkbox-- I could add a few more geography stuff or identifier searching (like for collector numbers) but then not so simple!

Look ok? @atrox10  @ccicero @cjconroy ? More? If ok, let's make it a preset so available for all and can be shared easily (right now it's just saved in my profile)

**Comment by cjconroy** [comment]
Fine with me

**Comment by ccicero** [comment]
I think this looks good. As I mentioned in our meeting yesterday, it would be even better if the "is tissue" dropdown was automatically set to "yes" if possible (yes, more hand-holding) but at least it's obvious here. 

Thanks!

**Comment by campmlc** [comment]
I agree - I would prefer if the "is tissue" dropdown was automatically checked.

**Comment by dustymc** [comment]
>would be even better if the "is tissue" dropdown was automatically set to "yes"

https://arctos.database.museum/search.cfm?sp=Tissue&is_tissue=1&autosubmit=false

>linked on our MVZ website

These are possibilities, not recommendations: 

* https://arctos.database.museum/search.cfm?sp=Tissue&is_tissue=1&autosubmit=false&guid_prefix=MVZ:Herp (profile, collection, and a pre-selected option), or 
* https://arctos.database.museum/search.cfm?guid_prefix=MVZ%3AArch%2CMVZ%3ABird%2CMVZ%3AEgg%2CMVZ%3AFish%2CMVZ%3AHerp%2CMVZ%3AHild%2CMVZ%3AImg%2CMVZ%3AMamm%2CMVZObs%3ABird%2CMVZObs%3AFish%2CMVZObs%3AHerp%2CMVZObs%3AMamm&is_tissue=1&sp=Tissue&autosubmit=false (profile, institution, and a pre-selected option)

> make it a preset

We can, but all that will really do is make the profile (not any other parameters) available from...<img width="242" alt="Screenshot 2024-12-11 at 07 53 31" src="https://github.com/user-attachments/assets/81649791-0b47-4e79-a1ad-e8315672c78e" /> ...and I think I'm hearing that the parameters are the important thing here.

>what it would look like:

You can use redirects to make URls look like nearly anything you want, see https://arctos.database.museum/uamart (or give me a pretty name and ugly URL)

>with the Tissue checkbox

Note that a parameter in the URL will make that available - eg here's the art preset but with the usually-nonexistent tissue flag checked: https://arctos.database.museum/search.cfm?sp=uam_art&guid_prefix=UAM:Art&is_tissue=1&autosubmit=false

Of note: That starting point can never return anything, assuming the art collection isn't cataloging things with tissues - it's very easy to lose people by pre-filling things that their next search term will never be used in conjunction with.

**Comment by dustymc** [comment]
See also https://github.com/ArctosDB/arctos/issues/7659
